src/runtime/vdom/vdom-render.ts

```diff
@@ -683,50 +688,61 @@ const updateFallbackSlotVisibility = (elm: d.RenderNode) => {
   }
 };
 
+/**
+ * Component-global information about nodes which are either currently being
+ * relocated or will be shortly.
+ */
 const relocateNodes: RelocateNodeData[] = [];
 
-const relocateSlotContent = (elm: d.RenderNode) => {
+/**
+ * Mark the contents of a slot for renaming via adding references to them to
+ * the {@link relocateNodes} data structure. The actual work of relocating them
+ * will then be handled in {@link renderVdom}.
+ *
+ * @param elm a render node whose child nodes need to be relocated
+ */
+const markSlotContentForRenaming = (elm: d.RenderNode) => {
```

Comment:
Did you mean for this to be `markSlotContentForRelocation` instead of "renaming"?

Reply:
haha sure did!